### DISCUSSION TITLE
Create standard 65-byte EDCSA signature

### DISCUSSION THREAD

**Discussion by ServantMaximus** [ProjectOpenSea/seaport#634] [Q&A] [answered]
I am trying to use the following api: https://docs.opensea.io/v2.0/reference/create-an-offer

One of the mandatory fields is: "signature"

According to the manual this should be a standard 65-byte EDCSA (or 64-byte EIP-2098, or an EIP-1271)

How can I create this 65-byte EDCSA from my wallet using preferable python?
Please point me to the right way if possible.

**Comment by ryanio** [comment, marked as answer]
You can use [EIP-712 ](https://eips.ethereum.org/EIPS/eip-712) along with the [type hashes derived here](https://github.com/ProjectOpenSea/seaport/blob/c30dd90272609677606f03f9c885466f15e891eb/contracts/lib/ConsiderationBase.sol#L137). If you are using web3.py it has eip-712 support.
